extensions/dacpac/src/wizard/dataTierApplicationWizard.ts

```diff
@@ -107,8 +108,16 @@ export class DataTierApplicationWizard {
 		}
 
 		this.model.serverId = this.connection.connectionId;
+		this.setPages();
 
-		this.wizard = azdata.window.createWizard(loc.wizardTitle);
+		this.wizard.generateScriptButton.hidden = true;
+		this.wizard.generateScriptButton.onClick(async () => await this.generateDeployScript());
+		this.wizard.doneButton.onClick(async () => await this.executeOperation());
+
+		this.wizard.open();
```

Comment:
What is the essence of this change? Is there some existing issues around the wizard creation?

Reply:
no this is just moving some code around so it's easier to test